lib/ex_doc/refs.ex

```diff
@@ -132,8 +135,26 @@ defmodule ExDoc.Refs do
     end
   end
 
-  defp visibility(:hidden), do: :hidden
-  defp visibility(_), do: :public
+  defguardp is_kind(kind) when kind in [:callback, :function, :module, :type]
+  defguardp no_docs?(doc) when doc == :none or (is_map(doc) and map_size(doc) == 0)
```

Comment:
I believe the second condition could be simplified to `doc == %{}`.

Reply:
Implemented. Thank you